exercises/python-1-a/README.md

```diff
@@ -0,0 +1,23 @@
+# Introduction
+
+This is Part 1 of our research into how people write Python differently.
```

Comment:
Can we standardise on this sentence-per-line README approach everywhere if we do it here pls?

Reply:
This renders in the same way as the ones that have no newlines. My personal preference is to keep things on the same line instead of these (ignored) newlines, to have it visually conform to what is actually rendered.

Reply:
Yeah - I think we need to decide on an Exercism standard and use it throughout the project. This PR almost certainly isn't the place for that though, so let's not worry about it here. I'll speak to Katrina and see if she has a hard preference, and if not, we can open it for a fun discussion! :)

Reply:
Very good to have standards. I noticed that GitHub's margin is <80 characters, which may be worth taking into account when making such a Markdown formatting standard.

Reply:
I've made the newlines consistent pending an outcome on this "fun" discussion.